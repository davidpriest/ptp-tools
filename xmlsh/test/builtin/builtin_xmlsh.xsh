# simple tests of xmlsh builtin command
# cannot test interactive subshells in a script

# if called with 1 arg then calling self
if [ $# -gt 0 ] ; then
	echo Called into self
	echo \$0 is $(xfile -b $0)
	echo \$* is $*
	exit 0;
fi 

xmlsh $0 Arg1 Arg2


echo In parent shell
xmlsh -c 'echo In sub shell ; exit 1'
echo Exit value is $?
exit 0