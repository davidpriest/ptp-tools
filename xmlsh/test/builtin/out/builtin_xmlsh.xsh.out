Called into self
$0 is builtin_xmlsh
$* is Arg1 Arg2
In parent shell
In sub shell
Exit value is 1
